src/ipinfo.erl

```diff
@@ -74,18 +77,22 @@ create(AccessToken) ->
 create(AccessToken, Settings) ->
     CountriesFile = get_config(countries, Settings,
         filename:join(code:priv_dir(ipinfo), ?DEFAULT_COUNTRY_FILE)),
+    EuCountriesFile = get_config(eu_countries, Settings,
+        filename:join(code:priv_dir(ipinfo), ?DEFAULT_EU_COUNTRY_FILE)),
     BaseUrl = get_config(base_url, Settings, ?DEFAULT_BASE_URL),
     Timeout = get_config(timeout, Settings, ?DEFAULT_TIMEOUT),
     CacheTtl = get_config(cache_ttl, Settings, ?DEFAULT_CACHE_TTL_SECONDS),
     case prepare_countries(CountriesFile) of
         {ok, Countries} ->
             {ok, Cache} = ipinfo_cache:create(CacheTtl),
+            {ok, EuCountries} = prepare_countries(EuCountriesFile),
```

Comment:
Couldn't this also fail like above? We'd have to do a separate case thing for that too.